buildscripts/README.md

```diff
@@ -0,0 +1,97 @@
+    * Run `nmake install`, or
+    * Copy the drivers:
+        * Find the directory where the newly compiled `php.exe` is
+        * Locate the compiled php_sqlsrv.dll and/or php_pdo_sqlsrv.dll 
+        * Copy the dll(s) to the `ext` subfolder  
+
+### Using the sample build scripts
+
+The sample build scripts, `builddrivers.py` and `buildtools.py`, are expected to build our extensions for PHP in Windows.
+
+#### Overview
+
+When asked to provide the PHP version, you should enter values like `7.1.7`. If it's alpha, beta, or RC, make sure the name you provide matches the PHP tag name without the prefix `php-`. For example, for PHP 7.2 beta 2, the tag name is `php-7.2.0beta2`, so you should enter `7.2.0beta2`. Visit [PHP SRC]( https://github.com/php/php-src) to find the appropriate tag names.
+
+It's recommended that the PHP SDK is unzipped into the shortest possible path, preferrably somewhere near the root drive. Therefore, this script will create a `php-sdk` folder in the C:\ drive. This `php-sdk` folder will remain unless you remove it yourself. For ongoing development, it's suggested you keep it around. The build scripts will handle updating the PHP SDK if new version is available. 
+
+#### Steps
+
+1. Launch a regular `cmd` prompt 
+
+2. Change to the directory where the Python scripts `builddrivers.py` and `buildtools.py` are
+
+3. Interactive mode: 
+    * Run `py builddrivers.py` to use the interactive mode. Use lower cases to answer the following questions:
+        * PHP Version (i.e. the version number like `7.1.7` or `7.2.0beta2`)
+        * 64-bit?
+        * Thread safe?
+        * Driver?
+        * Debug enabled?
+        * Download source from GitHub?
+    * For `yes/no` questions, you can simply hit `ENTER` key for `yes`. Other questions are self-explanatory.
+    
+4. Use Command-line arguments
+    * Run `py builddrivers.py -h` to get a list of options and their descriptions
+    * For example, 
+        * `py builddrivers.py -v=7.1.7 -a=x86 -t=ts -d=all -g=no`
+        * `py builddrivers.py --PHPVER=7.0.22 --ARCH=x64 --THREAD=nts --DRIVER=sqlsrv --GITHUB=yes`
+
+5. Based on the given configuration, if the script detects the presence of the PHP source directory, you can choose whether to rebuild, clean or superclean:
+    * `rebuild` if you have always used the same configuration (32 bit, thread safe, etc.)
+    * `clean` to remove previous builds (binaries) 
+    * `superclean` to remove the entire `php-<version>-src` directory, which is often unnecessary
+
+6. If you choose not to download from a GitHub repository, you will be asked to provide the full path to your local Source folder.
+
+7. If the compilation is successful, you will be given the option to rebuild or quit. 
+
+#### Troubleshooting
+
+When something goes wrong during the build, the log file will be launched (you can find the log files in `C:\php-sdk`). Otherwise, the log file will not be shown, and they remain in `C:\php-sdk` until you remove them manually.
```

Comment:
Not necessarily only when the build failed. It can be somewhere in the scripts that went wrong.